setup.py

```diff
@@ -48,6 +48,7 @@
         "qiskit ~= 0.41.0",
         "qiskit-ibm-runtime ~= 0.8.0",
         "qiskit-aer ~= 0.11.2",
+        "numpy",
```

Comment:
This should not be needed, installing pytket will install numpy as well. Or do you need a specific version?

Reply:
I'm not sure if it's good practice to omit direct dependencies from the dependency list just because you know it's a transitive dependency of another direct dependency.

By not giving a version, a good package manager (not sure about pip, but surely...) should then use the version constraints of the transitive dependency, in this case the constraints given by pytket.

But I can remove it if you prefer

Reply:
That is a good argument, I have checked that, pip should handle this.